src/Components/FieldCheckbox/FieldCheckbox.js

```diff
@@ -146,11 +155,12 @@ function FieldCheckbox({
       // normal checkbox
       const labelClasses = classNames('db', {
         red: isInvalid,
+        required,
       });
       return (
         <Block direction="column" className="ml-2">
           <label htmlFor={id} className={labelClasses}>
-            {label}
+            {labelSpan}
```

Comment:
why doesn't this use `<Label>`?

Reply:
I think it was don't this way because of the styles involved. Also the label in this case wraps all of the text, which would then add the required asterisk at the end of the text block